content/partials/shared/_channel_enumeration.textile

```diff
@@ -1,6 +1,6 @@
 This enumerates all active channels in the application. This is a paginated API following the same API conventions as other paginated APIs in the "Ably REST library":/rest.
 
-This API is intended for occasional use by your servers only; for example, to get an initial set of active channels to be kept up to date using the "channel lifecycle metachannel":realtime/channel-metadata#subscribing-to-metadata-events . It is heavily rate-limited: only a single in-flight channel enumeration call is permitted to execute at any one time. Further concurrent calls will be refused with an error with code @42912@.
+This API is intended for occasional use by your servers only; for example, to get an initial set of active channels to be kept up to date using the "channel lifecycle metachannel":/realtime/channel-metadata#subscribing-to-metadata-events . It is heavily rate-limited: only a single in-flight channel enumeration call is permitted to execute at any one time. Further concurrent calls will be refused with an error with code @42912@.
```

Comment:
> "channel lifecycle metachannel":/realtime/channel-metadata#subscribing-to-metadata-events

The target material was reorganized a while back, so that anchor no longer exists. I think in this case we can link to `/realtime/metachannels` here, as that is the page that discusses the channel lifecycle metachannel.

Reply:
Good spot thanks, must have missed checking that anchor.